### ISSUE TITLE
Unable to install extension from source

### ISSUE THREAD

**Issue by guillaumesmo** [phpbrew/phpbrew#549]
Hi,

I downloaded php 5.3.29 with the following command : 
```
phpbrew install 5.3.29 +default+dbs
phpbrew switch php-5.3.29
```

Now I want to install the mssql extension located in ~/.phpbrew/build/php-5.3.29/ext/mssql. 
```
$ phpbrew ext
...
Available local extensions:
...
[ ] mssql
...

$ phpbrew ext install mssql
Exception: Invalid package version: mssql @ 
Trace:

    0) PEARX\Channel->fetchPackageReleaseXml('mssql', 'stable')
    1) PhpBrew\Extension\Provider\PeclProvider->buildPackageDownloadUrl('stable')
    2) PhpBrew\Extension\ExtensionDownloader->download(PhpBrew\Extension\Provider\PeclProvider, 'stable')
    3) PhpBrew\Command\ExtensionCommand\InstallCommand->execute('mssql')
    4) call_user_func_array([PhpBrew\Command\ExtensionCommand\InstallCommand, 'execute'], ['mssql'])
    5) CLIFramework\CommandBase->executeWrapper(['mssql'])
    6) CLIFramework\Application->run(['/usr/bin/phpbrew', 'ext', 'install', 'mssql'])
    7) CLIFramework\Application->runWithTry(['/usr/bin/phpbrew', 'ext', 'install', 'mssql'])
    8) require('phar:///usr/bin/phpbrew/scripts/phpbrew-emb.php')
```

Going into the ~/.phpbrew/build/php-5.3.29/ext/mssql folder and building manually works, and I successfully loaded the module in PHP. 

Is there a way to prevent phpbrew from trying to download from the web?

**Comment by marcioAlmada** [comment]
We need to add this extension to the pecl provider bundled list https://github.com/phpbrew/phpbrew/blob/develop/src/PhpBrew/Extension/Provider/PeclProvider.php#L73

**Comment by marcioAlmada** [comment]
fixed with 013b834, thanks to @guillaumesmo!
